Index: git/source/util_api/ccsp_msg_bus/ccsp_message_bus.c
===================================================================
--- git.orig/source/util_api/ccsp_msg_bus/ccsp_message_bus.c
+++ git/source/util_api/ccsp_msg_bus/ccsp_message_bus.c
@@ -19,7 +19,6 @@
 #include <string.h>
 #include <pthread.h>
 #include <dbus/dbus.h>
-#include <dbus/dbus-mainloop.h>
 #include <ccsp_message_bus.h>
 #include "ccsp_base_api.h"
 #include "ccsp_trace.h"
@@ -37,8 +36,8 @@ const int NANOSEC_PER_MILLISEC = 1000000
 #endif
 
 // EXTERNAL
-extern void _dbus_connection_lock (DBusConnection *connection);
-extern void _dbus_connection_unlock (DBusConnection *connection);
+extern void dbus_connection_lock (DBusConnection *connection);
+extern void dbus_connection_unlock (DBusConnection *connection);
 
 extern CCSP_DEADLOCK_DETECTION_INFO deadlock_detection_info;
 extern int   CcspBaseIf_timeout_protect_plus_seconds;
@@ -205,7 +204,7 @@ add_watch
 {
     CData *cd = data;
     
-    return _dbus_loop_add_watch 
+    return dbus_loop_add_watch 
                (
                    cd->loop,
                    watch
@@ -227,7 +226,7 @@ remove_watch
 {
     CData *cd = data;
     
-    _dbus_loop_remove_watch 
+    dbus_loop_remove_watch 
         (
              cd->loop,
              watch 
@@ -259,7 +258,7 @@ add_timeout
 {
     CData *cd = data;
 
-    return _dbus_loop_add_timeout 
+    return dbus_loop_add_timeout 
                (
                    cd->loop,
                    timeout 
@@ -281,7 +280,7 @@ remove_timeout
 {
     CData *cd = data;
 
-    _dbus_loop_remove_timeout 
+    dbus_loop_remove_timeout 
         (
             cd->loop,
             timeout 
@@ -309,7 +308,7 @@ cdata_new
     cd->connection = connection;
     
     dbus_connection_ref (cd->connection);
-    _dbus_loop_ref (cd->loop);
+    dbus_loop_ref (cd->loop);
 
     return cd;
 }
@@ -323,7 +322,7 @@ cdata_free
     CData *cd = data;
 
     dbus_connection_unref (cd->connection);
-    _dbus_loop_unref (cd->loop);
+    dbus_loop_unref (cd->loop);
 
     dbus_free (cd);
 }
@@ -362,8 +361,8 @@ dispatch_status_func
 
     if (new_status != DBUS_DISPATCH_COMPLETE)
     {
-        while ( ! _dbus_loop_queue_dispatch (loop, connection))
-            _dbus_wait_for_memory ();
+        while ( ! dbus_loop_queue_dispatch (loop, connection))
+            dbus_wait_for_memory ();
     }
 }
 
@@ -521,7 +520,7 @@ ccsp_connection_setup
     // check dispatch status
     if (dbus_connection_get_dispatch_status (connection) != DBUS_DISPATCH_COMPLETE)
     {
-        if ( ! _dbus_loop_queue_dispatch (loop, connection))
+        if ( ! dbus_loop_queue_dispatch (loop, connection))
             goto NO_MEM;
     }
 
@@ -577,7 +576,7 @@ CCSP_Message_Bus_Loop_Thread
 )
 {
     DBusLoop *loop = (DBusLoop *)ptr;
-    _dbus_loop_run (loop); // main loop, does _ref and _unref
+    dbus_loop_run (loop); // main loop, does _ref and _unref
     return NULL;
 }
 
@@ -946,7 +945,7 @@ CCSP_Message_Bus_Init
         */
 
         // start the loop and connect threads, should be just one of each, even if count > 1
-        bus_info->connection[count].loop = _dbus_loop_new();
+        bus_info->connection[count].loop = dbus_loop_new();
         pthread_create
             (
                 &thread_dbus_loop, 
@@ -1078,7 +1077,7 @@ CCSP_Message_Bus_Exit
 
         if(bus_info->connection[i].loop )
         {
-            _dbus_loop_quit(bus_info->connection[i].loop);
+            dbus_loop_quit(bus_info->connection[i].loop);
         }
     }
 
@@ -1571,7 +1570,7 @@ CCSP_Message_Bus_Send_Str
     }
 
     // get reply
-    _dbus_connection_lock(conn);
+    dbus_connection_lock(conn);
     reply = dbus_pending_call_steal_reply(pcall);
 
     if( ! reply) 
@@ -1591,7 +1590,7 @@ CCSP_Message_Bus_Send_Str
         pthread_mutex_lock(&cb_data->count_mutex);
         dbus_pending_call_set_notify (pcall, ccsp_msg_check_resp_sync, (void *)cb_data, NULL);
 
-        _dbus_connection_unlock(conn);
+        dbus_connection_unlock(conn);
 
 #ifndef WIN32 
         gettimeofday(&now, NULL);
@@ -1630,7 +1629,7 @@ CCSP_Message_Bus_Send_Str
     }
     else
     {
-        _dbus_connection_unlock(conn);
+        dbus_connection_unlock(conn);
         ret = analyze_reply(message, reply, NULL);
     }
 
@@ -1707,7 +1706,7 @@ CCSP_Message_Bus_Send_Msg
         goto EXIT;
     }
 
-    _dbus_connection_lock(conn);
+    dbus_connection_lock(conn);
     reply = dbus_pending_call_steal_reply(pcall);
     if( ! reply)  
     {
@@ -1723,7 +1722,7 @@ CCSP_Message_Bus_Send_Msg
         if( ! cb_data)
         {
             CcspTraceError(("<%s>: cb_data malloc fail \n", __FUNCTION__));
-            _dbus_connection_unlock(conn);
+            dbus_connection_unlock(conn);
             ret = CCSP_Message_Bus_OOM;
             goto EXIT;
         }
@@ -1736,7 +1735,7 @@ CCSP_Message_Bus_Send_Msg
         pthread_mutex_lock(&cb_data->count_mutex);
         dbus_pending_call_set_notify (pcall, ccsp_msg_check_resp_sync, (void *)cb_data, NULL);
 
-        _dbus_connection_unlock(conn);
+        dbus_connection_unlock(conn);
 
 #ifndef WIN32 
         gettimeofday(&now, NULL);
@@ -1775,7 +1774,7 @@ CCSP_Message_Bus_Send_Msg
     }
     else
     {
-        _dbus_connection_unlock(conn);
+        dbus_connection_unlock(conn);
         ret = analyze_reply(message, reply, result);
     }
 
Index: git/source/ccsp/include/ccsp_message_bus.h
===================================================================
--- git.orig/source/ccsp/include/ccsp_message_bus.h
+++ git/source/ccsp/include/ccsp_message_bus.h
@@ -56,7 +56,6 @@
 #ifndef CCSP_MESSAGE_BUS_H
 #define CCSP_MESSAGE_BUS_H
 #include <dbus/dbus.h>
-#include <dbus/dbus-mainloop.h>
 
 /*
 notes: see readme.txt
